packages/lpa-questionnaire-web-app/src/routes/files.js

```diff
@@ -3,15 +3,26 @@ const filesController = require('../controllers/files');
 const reqFilesToReqBodyFilesMiddleware = require('../middleware/req-files-to-req-body-files');
 const fileValidationRules = require('../validators/files');
 const { validationErrorHandler } = require('../validators/validation-error-handler');
+const fetchExistingAppealReplyMiddleware = require('../middleware/fetch-existing-appeal-reply');
+const fetchAppealMiddleware = require('../middleware/fetch-appeal');
 
 const router = express.Router();
 
 router.post(
   '/upload',
-  [reqFilesToReqBodyFilesMiddleware('documents'), fileValidationRules()],
+  [
+    fetchAppealMiddleware,
+    fetchExistingAppealReplyMiddleware,
```

Comment:
this may well be a tendril of accidentally spending 2 days investigating why this didn't work with cypress in chrome. That was playing absolute hell with our session state and as i look at it now I'm not sure we -do- need those calls? It's also possible that there's some advantage to refreshing our understanding of the reply before we start PUTting whatever we had on the session straight into the API, but at face value it's more likely that i just didn't trust the content of the session when i made these changes and they can likely be reverted. if everything stays green it's fine..